inference-engine/src/mkldnn_plugin/mkldnn_infer_request.cpp

```diff
@@ -308,9 +295,27 @@ InferenceEngine::Blob::Ptr MKLDNNPlugin::MKLDNNInferRequest::GetBlob(const std::
         data = _outputs[name];
         checkBlob(data, name, false);
     }
+
     if (!data) {
         IE_THROW() << "Cannot find blob with name: " << name;
     }
+
+    // check if preprocess required, but still wasn't set
+    auto preProcessedInput = std::find_if(std::begin(_networkInputs), std::end(_networkInputs),
```

Comment:
Just wondering why you moved this code block to the end of the function?

Reply:
Fixed